libpromises/evalfunction.c

```diff
@@ -2202,7 +2212,7 @@ static FnCallResult FnCallGetIndicesClassic(EvalContext *ctx, ARG_UNUSED const P
 
 static FnCallResult FnCallGetIndices(EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
 {
-    const char *name_str = RlistScalarValue(finalargs);
+    const char *name_str = RlistScalarValueSafe(finalargs);
```

Comment:
Perhaps a shortform for `rlist->val.type == RVAL_TYPE_SCALAR` would be appropriate, like `RlistIsScalar()` or something like that.

This might be appropriate some of the other places where `RlistScalarValueSafe()` is used as well.